react-common/components/profile/UserPane.tsx

```diff
@@ -27,8 +29,12 @@ export const UserPane = (props: UserPaneProps) => {
 
     return <div className="profile-user-pane">
         <div className="profile-portrait">
-            { picture?.dataUrl ?
-                <img src={picture?.dataUrl} alt={pxt.U.lf("Profile Picture")} />
+            { picUrl ?
+                // Google user picture URL must have referrer policy set to no-referrer
+                <div dangerouslySetInnerHTML={{__html: pxt.BrowserUtils.imgTag(picUrl, {
```

Comment:
Alternatively, we could add a meta tag to change the referrer policy globally:
```
<meta name="referrer" content="no-referrer" />
```
But this will affect all `fetch` calls. I'm not sure what it will break.

Reply:
Open to suggestions here. I don't love this solution.

Reply:
Ah only saw this now but suggestion in my review below \\/